compose/docker-compose-fixturenet-eth.yml

```diff
@@ -21,10 +21,14 @@ services:
 
   fixturenet-eth-geth-2:
     hostname: fixturenet-eth-geth-2
+    environment:
+      RUN_STATEDIFF: "true"
```

Comment:
Let's prefix all environment variables we invent with some consistent prefix so we know they are ours. I think we are already using `CERC_` elsewhere.

Reply:
Hmm, that might deserve a separate PR, because it would apply to everything in fixturenet-eth.env.